draft-irtf-cfrg-vdaf.md

```diff
@@ -5554,22 +5554,54 @@ of operation, such as Poplar1.
-| `0x00000002`                 | Prio3SumVec           | VDAF | {{prio3sumvec}}           |
-| `0x00000003`                 | Prio3Histogram        | VDAF | {{prio3histogram}}        |
-| `0x00000004`                 | Prio3MultihotCountVec | VDAF | {{prio3multihotcountvec}} |
-| `0x00000005` to `0x00000FFF` | reserved for Prio3    | VDAF | n/a                       |
-| `0x00001000`                 | Poplar1               | VDAF | {{poplar1-inst}}          |
-| `0xFFFF0000` to `0xFFFFFFFF` | reserved              | n/a  | n/a                       |
-{: #codepoints title="Unique identifiers for (V)DAFs."}
-
-> TODO Add IANA considerations for the codepoints summarized in {{codepoints}}.
+IANA is requested to make one new registry:
+
+* DAF and VDAF Identifiers
+
+This registry should be created under the heading "Verifiable Distributed
+Aggregation Functions (VDAF)", and administered under the Specification Required
+policy {{!RFC8126}}.
+
+The "VDAF Identifiers" registry lists identifiers for Distributed Aggregation
+Functions (DAFs) and Verifiable Distributed Aggregation Functions (VDAFs). These
+identifiers are four-byte values, so the minimum possible value is 0x00000000
+and the maximum possible value is 0xffffffff.
+
+Template:
+
+* Value: The four-byte identifier for the DAF or VDAF
+* Scheme: The name of the DAF or VDAF
+* Type: Either "DAF" for a Distributed Aggregation Function or "VDAF" for a
+  Verifiable Distributed Aggregation Function
+* Reference: Where the algorithm is defined
+
+The initial contents of the registry are as follows:
+
+| Value                        | Scheme                   | Type | Reference                             |
+|:-----------------------------|:-------------------------|:-----|:--------------------------------------|
+| `0x00000000`                 | Reserved                 | n/a  | RFC XXXX                              |
+| `0x00000001`                 | Prio3Count               | VDAF | {{prio3count}} of RFC XXXX            |
+| `0x00000002`                 | Prio3Sum                 | VDAF | {{prio3sum}} of RFC XXXX              |
+| `0x00000003`                 | Prio3SumVec              | VDAF | {{prio3sumvec}} of RFC XXXX           |
+| `0x00000004`                 | Prio3Histogram           | VDAF | {{prio3histogram}} of RFC XXXX        |
+| `0x00000005`                 | Prio3MultihotCountVec    | VDAF | {{prio3multihotcountvec}} of RFC XXXX |
+| `0x00000006`                 | Poplar1                  | VDAF | {{poplar1-inst}} of RFC XXXX          |
+| `0xFFFF0000` to `0xFFFFFFFF` | Reserved for Private Use | n/a  | n/a                                   |
+{: #codepoints title="Verifiable Distributed Aggregation Function Identifiers Registry"}
+
+[[ RFC EDITOR: Please replace "RFC XXXX" above with the RFC number assigned to
+this document. ]]
+
+VDAF identifiers are used for domain separation, as described in
+{{the-domain-separation-tag-and-binder-string}}.  Domain separation guards
```

Comment:
It would probably be good to explicitly provide an anchor label on the referenced section heading, in case we tweak its wording.